packages/sdk/src/tests/assets/nft.test.ts

```diff
@@ -0,0 +1,246 @@
+import {beforeEach, describe, expect, it} from 'vitest'
+
+import {NFT, NOT_MINTED_NFT_ERROR_MESSAGE, UPDATE_MINTED_NFT_ERROR_MESSAGE} from '../../assets/nft'
+
+const expectedTokenId = {
```

Comment:
@feliperodriguess From a previous conversations, i asked to change the keys to the following shape:

```
 {
  hex,
  decimal,
  part: { chainId, tokenNumber }
}
```

Reply:
✅